erpnext/hr/doctype/driver/driver.py

```diff
@@ -3,7 +3,13 @@
 # For license information, please see license.txt
 
 from __future__ import unicode_literals
+import frappe
 from frappe.model.document import Document
 
 class Driver(Document):
-	pass
+
+	def validate(self):
+		if not self.user_id:
+			return
+
+		self.employee = frappe.db.get_value("Employee", {"user_id": self.user_id}) or None
```

Comment:
`get_value` defaults to `None` anyway

```suggestion
		self.employee = frappe.db.get_value("Employee", {"user_id": self.user_id})
```